estate_account/models/estate_property.py

```diff
@@ -0,0 +1,28 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def estate_property_button_sold(self):
```

Comment:
Same idea about the self being multiple records, in this case it's going to crash